src/init.cpp

```diff
@@ -478,6 +478,9 @@ std::string HelpMessage(HelpMessageMode mode) {
     strUsage += HelpMessageOpt("-timeout=<n>",
                                strprintf(_("Specify connection timeout in milliseconds (minimum: 1, default: %d)"),
                                          DEFAULT_CONNECT_TIMEOUT));
+    strUsage += HelpMessageOpt("-torsetup=<n>",
```

Comment:
Seem like we treat this as a boolean. So I think change `-torsetup=<n>` to `-torsetup` looking better.

Reply:
Yes you're right.